src/app/carbon-estimator-form/carbon-estimator-form.component.html

```diff
@@ -195,7 +195,16 @@
             name="monthlyActiveUsers"
             type="number"
             id="monthlyActiveUsers"
-            min="1" />
+            min="1"
+            aria-describedby="monthlyActiveUsersError" />
+          <p
+            *ngIf="monthlyActiveUsers?.invalid && (monthlyActiveUsers?.dirty || monthlyActiveUsers?.touched)"
```

Comment:
We're using the new @if syntax elsewhere for conditional elements so we might as well be consistent.